src/IIIFPresentation/Repository/PresentationContext.cs

```diff
@@ -115,11 +115,13 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     { cp.Id, cp.CustomerId, cp.ManifestId, cp.CanvasOriginalId, cp.CanvasOrder, cp.ChoiceOrder })
                 .IsUnique()
                 .HasFilter("asset_id is null");
```

Comment:
Personal/opinion - I find multi-line fluent method calls more difficult to read without a space between.